allennlp/tests/nn/util_test.py

```diff
@@ -282,6 +282,31 @@ def test_masked_mean(self):
         matrix_mean = util.masked_mean(matrix, mask, dim=1).data.numpy()
         assert_array_almost_equal(matrix_mean, numpy.array([[3.0, 0.5], [-1.5, -1.75]]))
 
+    def test_masked_flip(self):
+        tensor = torch.FloatTensor([[[6, 6, 6], [1, 1, 1], [2, 2, 2]], [[3, 3, 3], [4, 4, 4], [5, 5, 5]]])
+        solution = torch.FloatTensor([[[6, 6, 6], [0, 0, 0]], [[4, 4, 4], [3, 3, 3]]])
+        response = util.masked_flip(tensor, [1, 2])
+        self.assertEqual(solution.size(), response.size())
+        self.assertEqual(torch.sum(torch.abs(solution - response)), 0)
```

Comment:
I see other tests in this file using `assert_almost_equal` for tensor equality. Seems like a reasonable replacement for these two lines.